src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -64,10 +57,33 @@ type Field =
   | 'description'
   | 'image'
   | 'video'
+  | 'contactInfo'
   | 'priceInfo'
   | 'audience'
   | 'organizer';
 
+type TabConfig = {
+  eventKey: string;
+  title: string;
+  Component: ReactNode;
+  visible: boolean;
+  completed: boolean;
+};
+
+type TabTitleProps = InlineProps & {
+  title: string;
+  completed: boolean;
+};
+
+const TabTitle = ({ title, completed, ...props }: TabTitleProps) => {
+  return (
+    <Inline spacing={3} {...getInlineProps(props)}>
+      {completed && <Icon name={Icons.CHECK_CIRCLE} color="#48874a" />}
+      <Text>{title}</Text>
+    </Inline>
+  );
+};
+
```

Comment:
Is there a specific benefit of extracting this to a separate component? Feels like we're just creating an extra layer of abstraction for the sake of it here.

Reply:
I don't know the exact reason.
I'll have to check with @simon-debruijn